.eslintignore

```diff
@@ -0,0 +1,9 @@
+# Never lint node_modules
```

Comment:
This file could be removed. We have defined exclusion in `ignorePatterns` and these settings are redundant.

Reply:
Ok, fine.